signer/client/signer_trust.go

```diff
@@ -170,9 +176,8 @@ func (trust *NotarySigner) ListAllKeys() map[string]string {
 	return map[string]string{}
 }
 
-// CheckHealth checks the health of one of the clients, since both clients run
-// from the same GRPC server.
-func (trust *NotarySigner) CheckHealth(timeout time.Duration) error {
+// CheckHealthDeprecated is deprecated.
```

Comment:
Since this is deprecated and doesn't appear to be used elsewhere, can it be deleted?